driver/sql/postgres/eventstore_test.go

```diff
@@ -297,7 +301,15 @@ func TestEventStore_Load(t *testing.T) {
 				factory.EXPECT().CreateEventStream(gomock.AssignableToTypeOf(&sql.Rows{})).Return(expectedStream, nil).Times(1)
 
 				strategy := mockSQL.NewPersistenceStrategy(ctrl)
-				strategy.EXPECT().ColumnNames().Return(columns).AnyTimes()
+
+				if matcher != nil {
+					strategy.EXPECT().PrepareSearch(matcher).Return([]byte(" AND version > $2 AND version < $3"), []interface{}{}).Times(1)
```

Comment:
You are only returning the query part here but no parameters. Can you please also add return parameters so that we know there are passed later on.